src/library_fs.js

```diff
@@ -1335,6 +1335,14 @@ FS.staticInit();` +
       // just create the tmp dirs that reside in it commonly
       FS.mkdir('/dev/shm');
       FS.mkdir('/dev/shm/tmp');
+      
+      // setup file system with time zone information
+      const zone = '/usr/share/zoneinfo/' + Intl.DateTimeFormat().resolvedOptions().timeZone;
+      FS.mkdirTree(PATH.dirname(zone));
+      FS.createFile(PATH.dirname(zone), PATH.basename(zone));
+      FS.mkdir('/etc');
+      FS.symlink(zone, '/etc/localtime')
```

Comment:
This seems like it can/should be a separate change?

Reply:
sure we can split this into a follow up PR if you would like

Reply:
Yes please.. its not clear to me that this is something that all users would want to pay the (codesize) cost of.

Reply:
That's fair, for my case icu4c will first check TZ env variable and then if not set falls back to these disk locations. Since TZ is always being set in the FS locations are less important